rllib/core/learner/torch/torch_learner.py

```diff
@@ -177,12 +177,12 @@ def build(self) -> None:
         # TODO (Kourosh): Instead of using _TorchAccelerator, we should use the public
         # api in ray.train but allow for session to be None without any errors raised.
```

Comment:
I think this should work now, but the behavior of `get_device()` is not well-defined when used outside of a session.

Reply:
And by "should work now" I mean the behavior after this PR should be equivalent to the behavior before Antoni's PR.